packages/runtime/src/components/shared/react-player.ts

```diff
@@ -0,0 +1,7 @@
+import Player from 'react-player'
+
+import { esmCompat } from '../../utils/esm-compat'
+
+// Vite SSR/ESM compatibility; upgrading to the latest ESM-only version of
+// `react-player` removes the need for this workaround but breaks Jest
+export const ReactPlayer = esmCompat(Player)
```

Comment:
We really need to switch from Jest to vitest, but it's more than I can chew at the moment.